tuist/tuist
# Align the behaviour of `--path` in `tuist fetch` with the rest of the CLI

Resolves https://github.com/tuist/tuist/issues/5608

### Short description 📝

Enabling a smart search of the Dependencies.swift file for fetching using the `-p` flag from everywhere!

### How to test the changes locally 🧐

1. Run the TuistKitTests suite.
2. To use a Tuist build with this changes and test against the steps described in the issue:
- checkout this branch
- follow the steps in the issue above and see that they are resolved!
- Note for some reason building tuist from source does not recognize the ProjectDescriptionHelpers framework. As such comment out its import in the Dependencies.swift file (line 3) and either comment out line number 7 or bring the code from the ProjectDescriptionHelpers folder into the file
- After this modification biulding and running from source should work `swift build ProjectDescription && swift run tuist fetch -p fixtures/app_with_spm_dependencies/Features/FeatureOne`
- To generate the specific project run `swift run tuist generate -p fixtures/app_with_spm_dependencies/Features/FeatureOne`

### Contributor checklist ✅


### Reviewer checklist ✅
